features/openstack_testflow.feature

```diff
@@ -3,25 +3,25 @@ Feature: Test creation and deletion of resources
   Scenario Outline: Manage resources in OpenStack
     Given I connect to OpenStack
 
-    Then I should be able to create <router_quantity> routers
-    Then I should be able to create <network_quantity> networks
-    Then I should be able to create <subnet_quantity> subnets
-    Then I should be able to create <security_group_quantity> security groups
-    Then I should be able to create <security_group_rules_quantity> security group rules
-    Then I create <quantity_volumes> volumes
-
+   # Then I should be able to create <router_quantity> routers
```

Comment:
why is that commented? comment is for all of the comments in the file

Reply:
this was for testing reasons, as you work on the testflow again this is all outdated